packages/core/src/languages/solidity/types.ts

```diff
@@ -97,7 +97,7 @@ export interface CompilerOutputMetadata {
 
 export interface CompilerOutputContract {
   abi: Array<Fragment>
-  storageLayout: SolidityStorageLayout
+  storageLayout: SolidityStorageLayout | undefined
```

Comment:
When could this be undefined? Doesn't the logic in `readStorageLayout` ensure that it's not undefined?